pages/tokens/applying-tokens.en.mdx

```diff
@@ -3,15 +3,17 @@
 There are two ways how you can apply tokens to your selection:
 
 ### Default behaviour (left-click)
-When you left-click a token this token is being put on your selection (multiple layers selected are possible!). For certain tokens we assume defaults, such as for `Colors` we assume you want to apply `Fill`.
+When you left-click on a token, it is applied to your current selection (multiple layers can be selected). For certain tokens, we provide default assumptions. For example, with `Colors`, we assume that you want to apply the token as a `Fill`.
 
 ### Specifying what to apply (right-click)
-You can right-click tokens to specify what property should be set, such as in `Spacing` the `Left`, `Top`, `Right`, `Bottom` or `Gap` properties individually
+You can also right-click on tokens to specify a particular token property you would like to set such as `Left`, `Top`, `Right`, `Bottom`, or `Gap` properties in the case of Spacing; can also be set individually.  
+<img src='/token-property.png' alt="Select token properties - step 1" width="800"/>
```

Comment:
General recommendation for alt text would be to describe what we see in the image. This is useful for screen reader users so they can understand what's visible here. A rule of thumb I follow here is imagine you were describing this image to a friend who is blind